libtextsecure/sendmessage.js

```diff
@@ -1238,8 +1264,12 @@ MessageSender.prototype = {
     profileKey,
     options
   ) {
-    const me = textsecure.storage.user.getNumber();
-    const numbers = groupNumbers.filter(number => number !== me);
+    // We always assume that only primary device is a member in the group
+    const primaryDeviceKey =
+      window.storage.get('primaryDevicePubKey') ||
+      textsecure.storage.user.getNumber();
+    const numbers = groupNumbers.filter(number => number !== primaryDeviceKey);
+
```

Comment:
I think that was a bug. the same issue was already fixed on this line before
https://github.com/loki-project/session-desktop/blame/f9c13e4018c33a478171fbf63b2116ed69444c5f/libtextsecure/sendmessage.js#L1126